src/riak_cs_list_objects_ets_cache.erl

```diff
@@ -52,9 +52,15 @@
 
 -define(DICTMODULE, dict).
 
+-ifdef(namespaced_types).
+-type dictionary() :: dict:dict().
```

Comment:
Parameterized types like `dict:dict(ref(), timer())` and `dict:dict(timer(), ref())` ?